src/Traits/ReflectionClassLikeTrait.php

```diff
@@ -698,8 +699,16 @@ public function isInstance($object)
         }
 
         $className = $this->getName();
-
-        return $className === get_class($object) || is_subclass_of($object, $className);
+        $objClass = get_class($object);
```

Comment:
This makes sense... I should check if there's a way to turn off a specific issue check for a whole project.